src/libpriv/rpmostree-rpm-util.c

```diff
@@ -859,7 +846,7 @@ rpmostree_get_refsack_for_commit (OstreeRepo                *repo,
   RpmOstreeRefSack *ret = NULL;
   g_autofree char *tempdir = NULL;
   glnx_fd_close int tempdir_dfd = -1;
-  DnfSack *hsack; 
+  g_autoptr(DnfSack) hsack = NULL;
```

Comment:
This looks wrong to me - the ownership is transferred to the refsack, no?

If that's really intended, I'd add an explicit comment in the PR.